core/src/main/java/org/apache/cxf/attachment/AttachmentUtil.java

```diff
@@ -75,6 +75,9 @@
     private static final Random BOUND_RANDOM = new Random();
     private static final CommandMap DEFAULT_COMMAND_MAP = CommandMap.getDefaultCommandMap();
     private static final MailcapCommandMap COMMAND_MAP = new EnhancedMailcapCommandMap();
+    
+    private static final String ERROR_MSG = "The value set as ";
```

Comment:
this only a prefix of message. Is it possible to keep it as is? Which check report this?

Reply:
@amarkevich  . I will revert the changes . It is from SonarLint stating the below
**String literals should not be duplicated (java:S1192)**
Duplicated string literals make the process of refactoring error-prone, since you must be sure to update all occurrences.
On the other hand, constants can be referenced from many places, but only need to be updated in a single place.

I find it before was much better to read the code. Once we add constants String it interrupt the sequence of code reading .